src/script/lua_api/l_object.cpp

```diff
@@ -141,7 +141,9 @@ int ObjectRef::l_remove(lua_State *L)
 	UNORDERED_SET<int>::iterator it;
 	for (it = child_ids.begin(); it != child_ids.end(); ++it) {
 		ServerActiveObject *child = env->getActiveObject(*it);
-		child->setAttachment(0, "", v3f(0, 0, 0), v3f(0, 0, 0));
+		// Child can be NULL if it was deleted earlier
+		if (child)
```

Comment:
Please set ServerActiveObject *child = env->getActiveObject(*it) in this condition directly

else okay for me